make clean && make => build error

Make clean output

    awi@ttv-awi04 ~/Projects/gherkin-dotnet> make clean
    rm -rf .compared .built .run_tests acceptance Gherkin/Parser.cs
    rm -rf */bin
    rm -rf */obj
    rm -rf */packages
    rm -rf ./output

Make output

    awi@ttv-awi04 ~/Projects/gherkin-dotnet> make
    rm Gherkin/Parser.cs
    rm: Gherkin/Parser.cs: No such file or directory
    make: *** [Gherkin/Parser.cs] Error 1

Somehow the `Gherkin/Parser.cs` target is not called in the makefile.
@aslakhellesoy Any ideas where the error is? I am at the end with my make knowledge.

--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ clean:
 .PHONY: clean
 
 Gherkin/Parser.cs: gherkin.berp gherkin-csharp.razor berp/berp.exe
-	rm $@
+	rm -f $@
 	mono berp/berp.exe -g gherkin.berp -t gherkin-csharp.razor -o $@
 
 .sln_built_debug: Gherkin/Parser.cs $(CS_FILES) Gherkin/gherkin-languages.json

